package.json

```diff
@@ -0,0 +1,24 @@
+{
+  "devDependencies": {
+    "ts-node": "^10.9.1",
+    "typescript": "^5.1.6"
+  },
+  "dependencies": {
+    "@types/node": "^20.4.6",
+    "emojibase": "^15.0.0",
+    "emojibase-data": "^15.0.0",
+    "i": "^0.3.7",
```

Comment:
What is this?

Reply:
I *don't* know, ha. Will see if I can remove it.

Reply:
I'm guessing it was an accident of `yarn install i <blah>` where `i` is the short alt for `install`